velox/exec/Window.cpp

```diff
@@ -102,31 +102,32 @@ Window::Window(
   createWindowFunctions(windowNode, inputType);
 }
 
+VectorPtr const Window::createConstantForWindowArg(
```

Comment:
This doesn't need to be a member function. You can pass the pool() as a parameter and make this a local function to the file.

Reply:
Apologies for missing this, passing a pointer to the pool worked.